addons/godot-next/inspector_plugins/delegation_inspector_plugin.gd

```diff
@@ -0,0 +1,32 @@
+tool
+extends EditorInspectorPlugin
+
+var obj_stack: Array
+
+#warning-ignore:unused_argument
+func can_handle(p_object: Object) -> bool:
+	return true
+
+func parse_begin(p_object: Object) -> void:
+	obj_stack.push_front(p_object)
```

Comment:
haha, if you're gonna call it `obj_stack`, I wouldn't start right off the bat with `push_front()` when Stacks can only do `push_back()`, `back()`, and `pop_back()`. XD

Reply:
though Stack were pushed and pop from the front!